src/latex/preamble.tex

```diff
@@ -90,3 +90,7 @@
 \newcommand{\legendbox}[1]{%
   \textcolor{#1}{\rule{\fontcharht\font`X}{\fontcharht\font`X}}%
 }
+
+%% Change the following to adjust the spacing between items for all itemized and
+%% enumerated lists in the document.
+\newcommand\itmsep{1pt}
```

Comment:
There is a "standard" package for doing exactly this: https://ctan.org/pkg/enumitem?lang=en
```suggestion
```